infrastructure/src/test/java/org/corfudb/infrastructure/management/FailureDetectorTest.java

```diff
@@ -43,7 +43,7 @@ public void testPollRound() {
         );
         Duration time = Duration.ofMillis(System.currentTimeMillis() - start);
         assertThat(time).isGreaterThan(Duration.ofMillis(450));
-        assertThat(time).isLessThan(Duration.ofSeconds(2));
+        assertThat(time).isLessThan(Duration.ofSeconds(10));
```

Comment:
Not sure why this is needed. If it is for test stability then I would suggest its done in a separate PR.